app/src/components/main/grid/gridBody/gridRow/gridRow.jsx

```diff
@@ -52,25 +54,36 @@ export class GridRow extends Component {
     excludeFromSelection: [],
     gridRowClassName: '',
     level: 0,
+    isSomeRowHighlighted: false,
   };
 
   state = {
     withAccordion: false,
     expanded: false,
+    updateHighlight: true,
   };
 
   componentDidMount() {
     this.handleAccordion();
+
+    if (this.props.isSomeRowHighlighted && this.state.updateHighlight) {
+      // eslint-disable-next-line
+      this.setState({ updateHighlight: false });
```

Comment:
Looks like a hack. What is it for?

Reply:
In some cases, UI is not updated. So I need to trigger render here.